DTW/Bridge DTF/map.json

```diff
@@ -0,0 +1,98 @@
+{
+	"name": "Bridge DTF",
+	"authors": ["Clijmart"],
+	"version": "1.0",
+	"gametype": "DTM",
+	"teams": [
+		{
+			"id": "benny",
+			"name": "Benny",
+			"color": "yellow",
+			"min": 1,
+			"max": 50
+		},
+		{
+			"id": "lucas",
+			"name": "Lucas",
+			"color": "gold",
+			"min": 1,
+			"max": 50
+		}
+	],
+	"spawns": [
+		{"teams": ["spectators"], "coords": "0.5, 51, 0.5, 0"},
```

Comment:
One last thing, the last 0 isn't really needed on both maps. You don't have to put the yaw if it's 0.